upstream-community-operators/kubeflow/1.2.0/kubeflow.v1.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,287 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: >-
```

Comment:
this example should be updated to reflect one of the supported kfdef 1.2.0. for example, `tensorboard` is not part of the manifests.

Reply:
Done - tested with https://raw.githubusercontent.com/moficodes/manifests/master/kfdef/kfctl_openaihub_openshift_tekton.yaml